tests/cli/test_data_loader.py

```diff
@@ -0,0 +1,109 @@
+#
+# Copyright (c) 2019, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import unittest
+import uuid
+
+from collections import namedtuple
+from mock import MagicMock
+
+from cli.data_loader import DataLoader
+
+
+class TestDataLodaer(unittest.TestCase):
+
+    def test_get_run_qualified_name(self):
+        # given
+        exp = MagicMock()
+        exp.name = 'exp_name'
+        run_info = MagicMock()
+        run_info.name = None
+        run_info.run_uuid = 'run_uuid'
+
+        # then
+        # pylint: disable=protected-access
+        self.assertEqual(DataLoader._get_run_qualified_name(exp, run_info), "exp_name/run_uuid")
```

Comment:
Either rename `then` to `expect` or extract `DataLoader._get_run_qualified_name(exp, run_info)` to the `when` section.